framework/src/Volo.Abp.Features/Volo/Abp/Features/FeatureChecker.cs

```
@@ -37,7 +37,11 @@ public class FeatureChecker : FeatureCheckerBase
         
         public override async Task<string> GetOrNullAsync(string name)
         {
-            var featureDefinition = FeatureDefinitionManager.Get(name);
+            var featureDefinition = FeatureDefinitionManager.GetOrNull(name);
```

I think it should throw an exception if there is no feature definition. This may be by design.